chatt-app/Login.java

```diff
@@ -1,81 +0,0 @@
-import java.awt.event.ActionEvent;
-import java.awt.event.ActionListener;
-import java.awt.event.KeyEvent;
-import java.awt.event.KeyListener;
-import java.io.IOException;
-import java.net.UnknownHostException;
-
-import javax.swing.*;
-
-

```

Comment:
Description: A Login page for a chat app program that redirects the user to the chat client